snips_nlu/intent_classifier/log_reg_classifier.py

```diff
@@ -267,7 +267,7 @@ def log_best_features(self, top_n=50):
                 log += "\n{} -> {}".format(feature_name, feature_weight)
         return log
 
-    def log_activation_weights(self, text, x, top_n=50):
+    def log_activation_weights(self, text, x, top_n=400):
```

Comment:
Do we want to change the default value, or is it a remaining change from a debugging session ?

Reply:
yes debugging change